src/main/java/org/example/Phone.java

```diff
@@ -0,0 +1,20 @@
+package org.example;
+
+public interface Phone {
```

Comment:
since you can store members like batteryLife directly here, an abstract class is more suitable. Also, common implementations for all phones (eg - call(number)) can be implemented directly in the abstract class